features/circular_reference.feature

```diff
@@ -6,10 +6,12 @@ Feature: Circular references handling
   @createSchema
   @dropSchema
   Scenario: Create a circular reference
-    When I send a "POST" request to "/circular_references" with body:
+    When I add "Content-Type" header equal to "application/ld+json"
```

Comment:
Is it compulsory to send the `Content-Type` request header now? Or is there still a default fallback format?

If it's compulsory, then https://github.com/api-platform/core/blob/44e5fc0cb7bfdcc090086dcb53e4da4772c8a097/src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php#L91 needs to be updated?